app/choropleth-config/index.js

```diff
@@ -188,6 +170,24 @@ const choroplethConfigs = [
       '#e54444',
     ],
   },
+  {
+    group: 'Census',
+    id: 'pop65pl1',
```

Comment:
In issue 16067, this variable is referred to as `pop65pl`, not `pop65pl1`. I think Populations variable names add the `1` to the ACS version of the variable to differentiate it from the Decennial version

Reply:
Gocha, I wasn't aware of that. It can be a bit confusing without prior awareness.